airbyte-cdk/bulk/core/load/src/testFixtures/kotlin/io/airbyte/cdk/load/write/BasicFunctionalityIntegrationTest.kt

```diff
@@ -1684,6 +1793,79 @@ abstract class BasicFunctionalityIntegrationTest(
         )
     }
 
+    /** Run a sync in dedup mode with composite PK, then drop one of the PKs entirely. */
+    @Test
+    open fun testBizarrePkSchemaEvolution() {
```

Comment:
Does adding these tests here mean all destinations using the `BasicFunctionalityIntegrationTest` will start running these?

Reply:
it does. IMO that's half the point of putting these tests into the CDK, is that all connectors get them automatically (and we should be comfortable disabling them + writing a ticket to fix them if they cause breakage)